website/docs/interactors/4-write-your-own.md

```diff
@@ -69,40 +70,44 @@ actions: {
 }
 ```
 
-Or you could use destructuring to make this a bit shorter:
+Or you could use destructuring to make it a bit shorter:
 
 ```js
 actions: {
   click: ({ perform }) => perform(element => element.click());
 }
 ```
 
-The former syntax is necessary if you want to write an action that delegates to other interactors' actions. For example, imagine that you want to create a form interactor that has a submit action. You could take this approach:
+The former syntax is necessary if you want to write an action that delegates to the actions of other interactors. For example, imagine that you want to create a form interactor that has a submit action. You could take this approach:
 
 ```js
 import { Button, createInteractor } from 'bigtest';
 
 export const Form = createInteractor<HTMLFormElement>('form')({
   selector: 'form',
   actions: {
-    async submit(interactor){
-      await interactor.find(Button('Submit')).click();
-    }
+	async submit(interactor){
+  	  await interactor.find(Button('Submit')).click();
+	}
   }
```

Comment:
i like the 2-space tabbing and that's what we've been using everywhere else in the docs